lib/remote_system.rb

```diff
@@ -93,7 +99,8 @@ def run_command(*args)
   # Tries to run the noop-command(:) on the remote system as root (without a password or passphrase)
   # and raises an Machinery::Errors::SshConnectionFailed exception when it's not successful.
   def connect
-    LoggedCheetah.run "ssh", "-q", "-o", "BatchMode=yes", "#{remote_user}@#{host}", ":"
+    LoggedCheetah.run *build_basic_ssh_command, "-q", "-o", \
```

Comment:
Ambiguous splat operator. Parenthesize the method arguments if it's surely a splat operator, or add a whitespace to the right of the `*` if it should be a multiplication.